src/components/Input/components/Switch/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import InputBase from '../InputBase';
+import { Props } from './types';
+
+
+function Switch({ className, ...otherProps }: Props): ReactElement {
+    const switchClasses = clsx(
+        'brew-Input--Switch',
```

Comment:
`--` denotes a state or condition. In this case, I think it's safe to just use the class `brew-Switch` since we likely won't have any other component using that name.